Makefile

```diff
@@ -121,9 +126,67 @@ $(LIMA_DOWNLOAD_DIR)/$(LIMA_DEPENDENCY_FILE_NAME):
 .PHONY: download.lima-dependencies
 download.lima-dependencies: $(LIMA_DOWNLOAD_DIR)/$(LIMA_DEPENDENCY_FILE_NAME)
 
-.PHONE: install.lima-dependencies
+.PHONY: install.lima-dependencies
 install.lima-dependencies: download.lima-dependencies
```

Comment:
I'm on the fence about this because it really does take a _long time_. Kinda want to make it separate for now so it's easy to do it conditionally in finch's makefile. I want to move it into an async process (like how our macOS `lima-and-qemu.tar.gz` is created) to mitigate how long extracting the archive takes as a fast follow